src/example/loadfont.cpp

```diff
@@ -0,0 +1,36 @@
+#include <crepe/api/Text.h>
+#include <crepe/facade/Font.h>
+#include <crepe/facade/SDLContext.h>
+#include <crepe/facade/SDLFontContext.h>
+#include <crepe/manager/Mediator.h>
+#include <exception>
+#include <iostream>
+#include <memory>
+using namespace crepe;
+int main() {
+
+	SDLFontContext font_facade;
+	Mediator mediator;
+	SDLContext sdl_context{mediator};
+	try {
+		// Correct way to create a unique pointer for Text
+		std::unique_ptr<Text> label
+			= std::make_unique<Text>(1, vec2(100, 100), vec2(0, 0), "test test", "OpenSymbol");
+		std::unique_ptr<Asset> asset = font_facade.get_font_asset(label->font_family);
+		std::cout << "path: " << asset->get_path() << std::endl;
+		std::unique_ptr<Font> font = std::make_unique<Font>(*asset, mediator);
```

Comment:
Ik zou trouwens verwachten dat de constructor van Font ergens een exception gooit in het geval dat het gespecificeerde lettertype niet gevonden is of niet geladen kan worden. Nu kun je een ongeldige `Font` instantie hebben (= een die altijd NULL zal returnen bij `get_font`)

Reply:
bij mij wordt er dan een standaard font geladen, ik heb ik het example een font_family naam ingevult die zeker niet op mijn pc voorkomt maar je krijgt nogsteeds een geldige filepath terug

Reply:
oh en ik heb in de constructor een exception toegevoegd als het filepath invalid is